tests/utils/is-set.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+import * as t from '../../src/index';
+
+
+describe('isSet', () => {
+
+   it('correctly classifies sets', () => {
+      expect(t.isSet(new Set([]))).to.strictlyEqual(true);
+      expect(t.isSet(new Set([ 'a', 'b', 'c' ]))).to.strictlyEqual(true);
+      expect(t.isSet(new Set([ 4 ]))).to.strictlyEqual(true);
+      expect(t.isSet(new Set([ 'a', 'b', 'c', 4 ]))).to.strictlyEqual(true);
+      // eslint-disable-next-line
```

Comment:
This `eslint-disable` needed? If so, can we disable the specific rule that failing?

Reply:
Removed. Must've been necessary for the linting rules of 3 years ago, but not needed now.